ACD-824 (#43)
Actindo-AG/Documentation: FiegeIntegration/UserInterface/01_Connections.md

- ![Edit](../../Assets/Icons/Edit01.png "[Edit]") (Edit)  
    Click this button to edit the selected Fiege connection. This button is only displayed if a single checkbox in the list of connections is selected. Alternatively, you can click directly a row in the list to edit a connection. The *Edit connection* view is displayed, see [Edit Fiege connection](#edit-Fiege-connection). 

The list displays all available connections. Depending on the settings, the displayed columns may vary. All fields are read-only.

- *Name*  
    Connection name.

- *Status*  
    Connection status. The following statuses are available:
    - ![Status](../../Assets/Icons/Status01.png "[Status]") **Active**  
    The connection is enabled and data is being synchronized via the connection.
    - ![Status](../../Assets/Icons/Status04.png "[Status]") **Inactive**   
    The connection is disabled and no data is being synchronized via the connection.

- *Driver*  
    Driver name.

- *ID*  
    Connection identification number. The ID number is automatically assigned by the system.

- ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add)  

>   Click this button to add a Fiege connection. The *Create connection* view is displayed, see [Create Fiege connection](#create-Fiege-connection).

Author: Einrückung prüfen

- *Bearer token (fiegeExport)*  
    Enter the bearer token as provided by Fiege.

- [SAVE]  
    Click this button to save the connection. 

## Edit Fiege connection

*Fulfillment > Settings > Tab CONNECTIONS > Select Fiege connection*

![Edit Fiege connection](../../Assets/Screenshots/Fulfillment/Settings/Connections/FiegeIntegration/EditConnectionCredentials.png "[Edit Fiege connection]")

- ![Back](../../Assets/Icons/Back02.png "[Back]") (Back)   
    Click this button to close the *Edit connection* view and return to the connection list. All changes are rejected.

- *Name*   
    Connection name. Click the button ![Edit](../../Assets/Icons/Edit02.png "[Edit]") (Edit) to the right of the name to edit it.

- ![Edit](../../Assets/Icons/Edit02.png "[Edit]") (Edit)  
    Click this button to edit the connection name. 

- ![Check](../../Assets/Icons/Check.png "[Check]") (Apply)  
    Click this button to apply the changes to the connection name. This button is only displayed if you are editing the connection name.

- *Created DD/MM/YYYY*  

> Creation date of the connection. This field is read-only.

Author: Einrückung prüfen

## Edit Fiege connection &ndash; Credentials

*Fulfillment > Settings > Tab CONNECTIONS > Select Fiege connection > Tab Credentials*

![Edit Fiege credentials](../../Assets/Screenshots/Fulfillment/Settings/Connections/FiegeIntegration/EditConnectionCredentials.png "[Edit Fiege credentials]")

- *Driver*  

>     Driver name. For Fiege, *Fiege Driver* is displayed. This drop-down list is read-only.

Author: ```suggestion
    Driver name. For Fiege, the *Fiege Driver* option is displayed. This drop-down list is read-only.
```

**Credentials** 

 - *Bearer token (fiegeExport)*  
    Click the field to edit the bearer token.

- [SAVE]  

>   Click this button to save any changes made.

Author: Einrückung prüfen

## Edit Fiege connection &ndash; Settings

*Fulfillment > Settings > Tab CONNECTIONS > Select Fiege connection > Tab Settings*

![Edit Fiege settings](../../Assets/Screenshots/Fulfillment/Settings/Connections/FiegeIntegration/EditConnectionCredentials.png "[Edit Fiege settings]")

[comment]: <> (NoE Beispiel mit Settings genommen und anomymisiert. Evtl. ein anderes Screenshot?)

> By default, the *Settings* tab is composed of the following entries: 

Author: ```suggestion
The *Settings* tab is composed of the following setting entries: 
```
warum by default?

- [Warehouse ID fallback](#warehouse-id-fallback)
- [Bearer token](#bearer-token)
- [A Warehouse](#a-warehouse)
- [B Warehouse](#b-warehouse)
- [C Warehouse](#c-warehouse)

[comment]: <> (Evtl. fliegt der Bearer token aus den Settings raus. Wenn soweit, neue Screenshots für die Settings hinzufügen und Menüpunkt hier löschen.)

### Warehouse ID fallback

*Fulfillment > Settings > Tab CONNECTIONS > Select Fiege connection > Tab Settings > Menu entry Warehouse ID fallback*

![Warehouse ID fallback](../../Assets/Screenshots/Fulfillment/Settings/Connections/FiegeIntegration/EditConnectionSettings_WarehouseIDFallback.png "[Warehouse ID fallback]")

**Warehouse ID fallback**

- [Save]  
    Click this button to save any changes made.

- *Warehouse ID fallback*  

>     Enter the number of the warehouse to be used as fallback warehouse for stock movements if no other reference is found.

Author: ```suggestion
    Enter the ID of the warehouse to be used as fallback warehouse for stock movements if no other reference is found.
```

### Bearer token

*Fulfillment > Settings > Tab CONNECTIONS > Select Fiege connection > Tab Settings > Menu entry Bearer token*

![Bearer token](../../Assets/Screenshots/Fulfillment/Settings/Connections/FiegeIntegration/EditConnectionSettings_BearerToken.png "[Bearer token]")

[comment]: <> (Evtl. fliegt der Bearer token aus den Settings raus. Wenn soweit, neue Screenshots für die Settings hinzufügen und Menüpunkt hier löschen.)

**Bearer token**

- [Save]  
    Click this button to save any changes made.

- *Bearer token*  
    Enter the bearer token for authentication with the Fiege API.

### A Warehouse

*Fulfillment > Settings > Tab CONNECTIONS > Select Fiege connection > Tab Settings > Menu entry A Warehouse*

![A Warehouse](../../Assets/Screenshots/Fulfillment/Settings/Connections/FiegeIntegration/EditConnectionSettings_AWarehouse.png "[A Warehouse]")

**A Warehouse**  

- [Save]  
    Click this button to save any changes made.

- *Warehouse ID A*  

>     Enter the number of the warehouse to be used as A warehouse.

Author: ```suggestion
    Enter the ID of the warehouse to be used as A warehouse.
```

### B Warehouse

*Fulfillment > Settings > Tab CONNECTIONS > Select Fiege connection > Tab Settings > Menu entry B Warehouse*

![B Warehouse](../../Assets/Screenshots/Fulfillment/Settings/Connections/FiegeIntegration/EditConnectionSettings_BWarehouse.png "[B Warehouse]")

**B Warehouse**

- [Save]  
    Click this button to save any changes made.

- *Warehouse ID B*  

>     Enter the number of the warehouse to be used as B warehouse.

Author: ```suggestion
    Enter the ID of the warehouse to be used as warehouse B.
```

### C Warehouse

*Fulfillment > Settings > Tab CONNECTIONS > Select Fiege connection > Tab Settings > Menu entry C Warehouse*

![C Warehouse](../../Assets/Screenshots/Fulfillment/Settings/Connections/FiegeIntegration/EditConnectionSettings_CWarehouse.png "[C Warehouse]")

**C Warehouse**

- [Save]  
    Click this button to save any changes made.

- *Warehouse ID C*  

>     Enter the number of the warehouse to be used as C warehouse.

Author: ```suggestion
    Enter the ID of the warehouse to be used as warehouse C.
```
